packages/datastore-lock/src/datastore-lock.ts

```diff
@@ -207,3 +207,56 @@ export class DatastoreLock {
     else return false;
   }
 }
+
+/**
+ * Parameters for creating a DatastoreLock.
+ */
+export interface DatastoreLockDetails {
+  lockId: string;
```

Comment:
@chingor13 What doc comment should I add for `lockId` and `target`?  There are no existing comments describing them.  I'm not sure of their purpose.

Reply:
I think I figured it out.